yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -704,44 +703,37 @@ export class Sequencer {
    */
   protected async getChainTip(): Promise<{ blockNumber: number; archive: Fr } | undefined> {
     const syncedBlocks = await Promise.all([
-      this.worldState.status().then((s: WorldStateSynchronizerStatus) => {
-        return {
-          number: s.syncSummary.latestBlockNumber,
-          hash: s.syncSummary.latestBlockHash,
-        };
-      }),
+      this.worldState.status().then(({ syncSummary }) => ({
```

Comment:
smells like a further unification of interfaces may be in order

Reply:
Definitely